shared/strings_util.h

```diff
@@ -8,6 +8,34 @@
 #include <algorithm>
 #include <filesystem>
 
+#include "../CompilerSource/OS_Switchboard.h"
```

Comment:
I don't think we should be pulling this from CompilerSource. Either just #ifdef WIN32 or move this file to shared if you really need it in shared.